source/Mission.cpp

```diff
@@ -282,7 +284,8 @@ void Mission::Load(const DataNode &node)
 				{"defer", DEFER},
 				{"visit", VISIT},
 				{"stopover", STOPOVER},
-				{"waypoint", WAYPOINT}
+				{"waypoint", WAYPOINT},
+				{"daily", DAILY}
```

Comment:
Add a comma at the end so that if we add another trigger we don't have to add it here again.

Reply:
done